dmoj/cli.py

```diff
@@ -168,8 +175,8 @@ class SubmitCommand(Command):
 
     def _populate_parser(self):
         self.arg_parser.add_argument('problem_id', help='id of problem to grade')
-        self.arg_parser.add_argument('language_id', help='id of the language to grade in (e.g., PY2)')
         self.arg_parser.add_argument('source_file', nargs='?', help='path to submission source (optional)')
+        self.arg_parser.add_argument('-lan', '--language-id', type=str, default='unknown', help='id of the language to grade in (e.g., PY2)')
```

Comment:
I don't know about you, but I certainly don't want to type `submit <problem> -lan PY2`. I want to type in `submit <problem> PY2` like before to type my code into `$EDITOR`. Not everyone is as masochistic as you are.